content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -42,6 +42,22 @@ Parameters are key/value pairs saved in structured files. Unlike stage-level
 are defined at the file level and include all parameters in the file. See
 `dvc params` for more information and how to compare between experiments.
 
+## Artifacts
+
+This section contains one or more artifact you want to declare in order to build
+a [Model Registry](/doc/use-cases/model-registry), Data Registry or more generic
+Artifact Registry.
+
+```yaml
+artifacts:
+  cv-classification:
+    path: models/cv.pkl
+    type: model
+```
```

Comment:
@dberenbaum does it make sense to show a compete example here? I.e. add labels, desc, meta, WDYT?

Reply:
Yes, let's show an example with all the fields. We also need a reference here of all the artifact fields (see https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#stage-entries or https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#available-configuration-fields), and we need to mention whatever rules we have for artifact names.